### PR TITLE
modify comment in core/v1 package

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
- fix core/v1 package comment typo
#### Which issue(s) this PR fixes:

Fixes # None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Comment by googs1025** [comment]
/remove-kind api-change
just typo fix

File: staging/src/k8s.io/api/core/v1/types.go [hunk 1]
```diff
@@ -3237,7 +3237,7 @@ const (
 	// during scheduling, for example due to nodeAffinity parsing errors.
 	PodReasonSchedulerError = "SchedulerError"
 
-	// TerminationByKubelet reason in DisruptionTarget pod condition indicates that the termination
+	// PodReasonTerminationByKubelet reason in DisruptionTarget pod condition indicates that the termination
```
Comment (liangyuanpeng) [inline staging/src/k8s.io/api/core/v1/types.go:3240]:
```suggestion
	// PodReasonTerminationByKubelet in DisruptionTarget pod condition indicates that the termination
```

Comment (liangyuanpeng) [reply staging/src/k8s.io/api/core/v1/types.go:3240]: Maybe this not need to change.

Comment (googs1025) [reply staging/src/k8s.io/api/core/v1/types.go:3240]: It's better to be consistent

**Comment by googs1025** [comment]
@deads2k /PTAL thanks

**Comment by googs1025** [comment]
@thockin can you help this? :)

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**